### DISCUSSION TITLE
Refiner possibility?

### DISCUSSION THREAD

**Discussion by jj05690569** [runew0lf/RuinedFooocus#93] [Ideas]
Hi! Just wanted to start off by saying this is such an improvement over original Fooocus, and made the entire experience so much more enjoyable over A1111. 
Is there any future plans of adding the refiner feature? specifically to be able to use SD1.5 models, like original Fooocus. Thanks!

**Comment by yownas** [comment]
No plans, and will probably not happen.

We even used to have the Refiner but removed it since it didn't add much to the images and just added complexity.

The philosophy behind RF is to have a simple but still useful interface to make pretty images. Trying to compete with other webui's to have the most options or advanced features is something that wouldn't make anyone happy. I rather make a simple interface that is fun to use, even if I can't do "everything".
